entity-types/infra-apacheserver/golden_metrics.yml

```diff
@@ -7,6 +7,12 @@ requestsPerSecond:
       from: ApacheSample
       eventId: entityGuid
       eventName: entityName
+    # defaulting to newRelic query for now since we cannot match value functions for OpenTelemetry cumulative metrics.
```

Comment:
I would prefer to not have commented code, maybe just leave a comment like no custom query for openTelemetry because we cannot match...
Also I've doubts about why it's commented, is the average(net.requestPerSecond) available when ingesting from OTEL?

Reply:
I have removed the commented code as suggested.